pom.xml

```diff
@@ -1870,5 +1870,39 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>ci</id>
+            <properties>
+                <!-- the *local* master, not refs/remotes/... -->
+                <gib.referenceBranch>master</gib.referenceBranch>
+            </properties>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>com.vackosar.gitflowincrementalbuilder</groupId>
+                        <artifactId>gitflow-incremental-builder</artifactId>
+                        <version>3.14.5</version>
+                        <extensions>true</extensions>
+                        <configuration>
+                            <disableIfBranchMatches>master</disableIfBranchMatches>
+                            <compareToMergeBase>true</compareToMergeBase>
+                            <uncommitted>true</uncommitted>
+                            <untracked>true</untracked>
+                            <excludePathsMatching>(\.github([/\\].*)?|\.mvn([/\\].*)?|mvnw|README.md|LICENSE)</excludePathsMatching>
```

Comment:
I believe a change in mvnw or maven settings in `.mvn` should result in all tests being run

Why are they safe to ignore?

Reply:
I removed them from here.